supervisor/utils/dbus.py

```diff
@@ -103,6 +103,8 @@ async def call_dbus(
             return await getattr(proxy_interface, method)(*args)
         except DBusError as err:
             raise DBus.from_dbus_error(err)
+        except (EOFError, OSError) as err:
```

Comment:
You can also get `BrokenPipeError`

Reply:
It appears we have users stuck on update indefinitely due to the raucstate issue. Given how bad an error here can be I think it might be better to just do a catch `Exeception`, re-raise it as a supervisor error and send it to sentry. That should limit the impact of future issues here and we can still review them.